fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentModelTemplate.java

```diff
@@ -143,6 +143,11 @@ protected void addGeneratedImport(Set<String> imports) {
     protected void addGeneratedAnnotation(JavaContext classBlock) {
     }
 
+    @Override
+    protected boolean isPartialUpdateSupported() {
+        return false;
```

Comment:
Similar question as in Androidgen, do we even want to disable this? It is a bit more overhead during generation but will simplify logic between generators.

Reply:
The reason is that MGMT doing lots of interface as fluent code. That is not supported by our current partial-update logic.
E.g. https://github.com/Azure/autorest.java/blob/eba2861394f41f3b16f22ea18521960f35287f72/typespec-tests/src/main/java/com/cadl/armresourceprovider/models/TopLevelArmResource.java

So, at present, MGMT is a full overwrite (with all generated code deleted, then regen).